airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -40,6 +40,7 @@
 
 class SlackStream(HttpStream, ABC):
     url_base = "https://slack.com/api/"
+    primary_key = None
```

Comment:
for messages and threads streams the pk is the `ts` , for channels it is ID

Reply:
Done

Reply:
reminder to add the primary key for all streams

Reply:
Done